X2PO/mar2024/galacticrepublic.json

```diff
@@ -0,0 +1,1727 @@
+{
+    "ARC-170 Starfighter": {
+        "sinker": {
+            "name": "\u201cSinker\u201d",
```

Comment:
I know the script I sent you created these (by converting to utf8). But it's better to have them in ascii format directly.
Could you alter the script to create that?

Reply:
I can do it not sooner than Wednesday. I see that AMG and XWA points also have UTF-8, should be ok for now?

Reply:
Done